packages/components/src/Button/GenericButton/GenericButton.tsx

```diff
@@ -47,7 +47,7 @@ export type GenericProps = {
   classNameOverride?: string
 }
 
-export type ButtonType = "submit" | "reset" | "button"
+export type GenericButtonType = "submit" | "reset" | "button"
```

Comment:
This is actually the HTML attribute values :o

```suggestion
export type ButtonType = "submit" | "reset" | "button"
```

Reply:
Actually, this never gets used... Just kill it.